clusterloader2/api/types.go

```diff
@@ -0,0 +1,83 @@
+package api
+
+import (
+	"time"
+)
+
+type Config struct {
```

Comment:
Please add struct-level comments to all types in this file.

Reply:
done